src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label for="surname"> Surname:</label>
```

Comment:
Do you need this space in text? Fix it everywhere
```suggestion
          <label for="surname">Surname:</label>
```